pkg/console/operator/sync_v400.go

```diff
@@ -78,29 +80,31 @@ func sync_v400(co *ConsoleOperator, consoleConfig *v1alpha1.Console) (*v1alpha1.
 	}
 	toUpdate = toUpdate || depChanged
 
-	// handy debugging block
-	logrus.Printf("sync_v400 complete. updates made:")
-	logrus.Printf("service changed: %v \n", svcChanged)
-	logrus.Printf("route is new: %v \n", rtChanged)
-	logrus.Printf("configMap changed: %v \n", cmChanged)
-	logrus.Printf("secret changed: %v \n", secChanged)
-	logrus.Printf("oauth changed: %v \n", oauthChanged)
-	logrus.Printf("deployment changed: %v \n", depChanged)
-
 	// if any of our resources have changed, we should update the consoleConfig.Status. otherwise, skip this step.
 	if toUpdate {
-		logrus.Infof("Sync_v400: To update Spec: %v", toUpdate)
+		logrus.Infof("sync_v400: to update spec: %v", toUpdate)
 		// TODO: set the status.
 		// setStatus(consoleConfig.Status, svc, rt, cm, dep, oa, sec)
 	}
 
+	defer func() {
+		logrus.Printf("sync loop 4.0.0 complete:")
+		logrus.Printf("service changed: %v", svcChanged)
```

Comment:
Since those are listed items, maybe add tab:
```go
logrus.Printf("\t service changed: %v", svcChanged)
```

maybe even without the space

Reply:
Prefer the `tab`?

<img width="864" alt="screen shot 2018-12-12 at 4 11 57 pm" src="https://user-images.githubusercontent.com/280512/49899096-b98d6b00-fe28-11e8-9f3f-9eba9b040c8d.png">

Reply:
Does it look odd with the `INFO[123]` far left?

Reply:
yeah, not really sure. Its definitely more clear that its an item fro ma list :)

Reply:
Works for me.

<img width="824" alt="screen shot 2018-12-13 at 10 42 50 am" src="https://user-images.githubusercontent.com/280512/49949548-e347a000-fec3-11e8-9090-3200e3fb2547.png">